docs/content/docs/data-structures/overview.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Overview of Distributed Data Structures
+menuPosition: 1
+---
+
+The Fluid Framework provides developers with distributed data structures (DDSes) that automatically ensure that each
+connected client has access to the same state. The APIs provided by DDSes are designed to be familiar to programmers
+who’ve used common data structures before.
+
+When using a DDS, you can largely treat it as a local object. You can add data to it, remove data, update it, etc.
+However, a DDS is not just a local object. A DDS can also be changed by other users that are editing.
+
+Choosing the correct data structure for your scenario can improve the performance and code structure of your application.
+Below we've enumerated the data structures and described when they may be most useful.
+
+## Key-value Data
+
+These DDSes are used for storing key-value data. They are optimistic and use a last-writer-wins merge policy.
+
+* [SharedMap]({{< relref "/docs/apis/map/sharedmap.md" >}}) - a basic key-value data structure
+* [SharedDirectory]({{< relref "/docs/apis/map/shareddirectory.md" >}}) – a SharedMap with hierarchical paths instead of
+simple keys
+* [SharedCell]({{< relref "/docs/apis/cell/sharedcell.md" >}}) – a “single-object SharedMap”; useful for wrapping objects.
+
+### Key Value Scenarios
+
+Key-value data structures are the most common choice for many scenarios.
+
+* user preference data
+* current state of a survey
+* the configuration of a view
+
+### Common Issues
+
+* Storing a counter in a map will have unexpected behavior. Use the SharedCounter instead.
+* Storing a lot of data in one key-value entry may cause performance issues. Each update will update the entire value.
+Try splitting the data across multiple keys
+* Storing arrays, lists, or logs in a key-value entry may lead to unexpected behavior because users can't collaboratively
+modify parts of one entry. Try storing the array or list data in a SharedSequence or SharedInk
+
+## Sequences
+
+These DDSes are used for storing sequential data. They are optimistic.
+
+* [SharedNumberSequence]({{< relref "SharedNumberSequence" >}}) – a sequence of numbers.
+* [SharedObjectSequence]({{< relref "/docs/apis/sequence/sharedobjectsequence.md" >}}) – a sequence of objects.
+* [SharedMatrix]({{< relref "SharedMatrix" >}}) – a data structure to efficiently use two-dimensional tabular data.
+* [SharedString]({{< relref "SharedString" >}}) – a specialized data structure for handling collaborative text.
+
+Sequence Data Structures are useful when you'll need to insert data in the middle of a list or array. Unlike the
+key-value data structures, sequences have a sequential order and can handle simultaneous inserts from multiple users.
+
+### Sequence Scenarios
+
+* Text editors
+* Tabular data
```

Comment:
Why not SharedMatrix?

Reply:
SharedMatrix is a sequence scenario?

Reply:
Well, when I think tabular data, I don't think sequence. Wouldn't SharedMatrix be a better fit for tabular data?